packages/log4j2-hotpatch/log4j2-hotpatch.spec

```diff
@@ -0,0 +1,36 @@
+%global debug_package %{nil}
+
+%global project hotpatch-for-apache-log4j2
+
+Name: %{_cross_os}log4j2-hotpatch
+Version: 1.3.0
+Release: 1%{?dist}
+Summary: Tool for hot patching log4j2 vulnerabilities
+License: Apache-2.0
+URL: https://github.com/corretto/%{project}
+Source0: https://github.com/corretto/%{project}/archive/%{version}/%{version}.tar.gz#/%{project}-%{version}.tar.gz
```

Comment:
Nit: left over comment at the end? Or is this intentional?

Reply:
It's intentional; it's a URL fragment that `rpm` will parse as the file name for use in the `%setup` macro.